pkg/filesystem/subvolume.go

```diff
@@ -93,14 +97,40 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 						continue
 					}
 					status = "in-use"
+				} else {
+					// check if the stale subvolume has snapshots.
+					if checkSnapshot(ctx, clientsets, operatorNamespace, clusterNamespace, fs.Name, sv.Name, svg.Name) {
+						status = "stale and has snapshot"
```

Comment:
can we have some pre-defined states in CLI output? the current one looks to be of more detailed one.

Reply:
what pre-defined states do you mean? the one in the subvolume info just says it's completed in this state. We are displaying the detailed state as the user could know that the subvolume has snapshot before deleting it.

Reply:
You need to have some const for it like below

```go

const (
InUse = "In-Use"
Stale = "Stale"
...
) and use it

Reply:
ohh okay, misunderstood, My bad. Cool will update it.